src/styles/app/03-app-shell.css

```
@@ -6,6 +6,10 @@
 	--app-frame-block: 0px;
 	--app-radius: 0px;
 	--window-control-toggle-offset: 90px;
+	--collapsed-sidebar-top-reserve: calc(
+		var(--window-control-toggle-offset) +
```

Stylelint (17.10.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)